src/portal/src/i18n/lang/fr-fr-lang.json

```diff
@@ -383,46 +383,46 @@
         "NEVER_EXPIRED": "Ne jamais expirer",
         "NAME_PREFIX": "Préfixe du nom du compte robot",
         "NAME_PREFIX_REQUIRED": "Le préfixe du nom du compte robot est obligatoire",
-        "UPDATE": "Update",
-        "AUDIT_LOG": "Audit Log",
-        "PREHEAT_INSTANCE": "Preheat Instance",
-        "PROJECT": "Project",
-        "REPLICATION_POLICY": "Replication Policy",
-        "REPLICATION": "Replication",
-        "REPLICATION_ADAPTER": "Replication Adapter",
-        "REGISTRY": "Registry",
-        "SCAN_ALL": "Scan All",
-        "SYSTEM_VOLUMES": "System Volumes",
-        "GARBAGE_COLLECTION": "Garbage Collection",
-        "PURGE_AUDIT": "Purge Audit",
+        "UPDATE": "Mettre à jour",
+        "AUDIT_LOG": "Log d'audit",
+        "PREHEAT_INSTANCE": "Préchauffer l'instance",
+        "PROJECT": "Projet",
+        "REPLICATION_POLICY": "Politique de réplication",
+        "REPLICATION": "Réplication",
+        "REPLICATION_ADAPTER": "Adapter de réplication",
+        "REGISTRY": "Registre",
+        "SCAN_ALL": "Scanner tout",
+        "SYSTEM_VOLUMES": "Volumes système",
+        "GARBAGE_COLLECTION": "Purge",
+        "PURGE_AUDIT": "Audit de purge",
         "JOBSERVICE_MONITOR": "Job Service Monitor",
-        "TAG_RETENTION": "Tag Retention",
+        "TAG_RETENTION": "Rétention de tag",
         "SCANNER": "Scanner",
         "LABEL": "Label",
-        "EXPORT_CVE": "Export CVE",
-        "SECURITY_HUB": "Security Hub",
-        "CATALOG": "Catalog",
-        "METADATA": "Project Metadata",
-        "REPOSITORY": "Repository",
-        "ARTIFACT": "Artifact",
+        "EXPORT_CVE": "Exporter CVE",
+        "SECURITY_HUB": "Tableau de bord de sécurité",
+        "CATALOG": "Catalogue",
+        "METADATA": "Métadonnées projet",
+        "REPOSITORY": "Dépôt",
+        "ARTIFACT": "Artefact",
         "SCAN": "Scan",
         "TAG": "Tag",
-        "ACCESSORY": "Accessory",
-        "ARTIFACT_ADDITION": "Artifact Addition",
-        "ARTIFACT_LABEL": "Artifact Label",
-        "PREHEAT_POLICY": "Preheat Policy",
-        "IMMUTABLE_TAG": "Immutable Tag",
+        "ACCESSORY": "Accessoire",
+        "ARTIFACT_ADDITION": "Artefact Addition",
```

Comment:
I think we should translate "addition" too but I don't understand the mean of this

Reply:
I don't know what this is, so I didn't change it.